src/chunk.c

```diff
@@ -2723,11 +2723,12 @@ ts_chunks_in(PG_FUNCTION_ARGS)
 	pg_unreachable();
 }
 
-/* has chunk, specifieid by chunk_id, been compressed */
+/* Check if this chunk can be compressed, that is is not dropped and has not
+ * already been compressed. */
 bool
-ts_chunk_has_associated_compressed_chunk(int32 chunk_id)
+ts_chunk_can_be_compressed(int32 chunk_id)
 {
-	bool compressed = false;
+	bool can_be_compressed = false;
```

Comment:
I think this new variable name is confusing since this function checks if the chunk is compressed, not whether it can be compressed.

Reply:
Now the semantics should be right.